front-end/src/ui-components/Card.tsx

```diff
@@ -19,10 +19,10 @@ const Card = ({ className, children }:Props ) => {
 export default styled(Card)`
 	background-color: white;
 	padding: 2rem 3rem 2rem 3rem;
-	border-style: solid;
```

Comment:
Why removing those?

Reply:
Using box shadow instead as shown in @erler mocks. Box shadow looks pretty.